[2022-06-27 13:33:31,623] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.accurate scheduled__2022-06-22T00:00:00+00:00 [queued]>
[2022-06-27 13:33:31,648] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.accurate scheduled__2022-06-22T00:00:00+00:00 [queued]>
[2022-06-27 13:33:31,660] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:33:31,661] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:33:31,662] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:33:31,673] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): accurate> on 2022-06-22 00:00:00+00:00
[2022-06-27 13:33:31,683] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'accurate', 'scheduled__2022-06-22T00:00:00+00:00', '--job-id', '895', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmp34xzp_9e', '--error-file', '/tmp/tmp7757f2fs']
[2022-06-27 13:33:31,687] {standard_task_runner.py:80} INFO - Job 895: Subtask accurate
[2022-06-27 13:33:31,679] {standard_task_runner.py:52} INFO - Started process 58650 to run task
[2022-06-27 13:33:31,802] {task_command.py:370} INFO - Running <TaskInstance: dag1.accurate scheduled__2022-06-22T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:33:31,916] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=accurate
AIRFLOW_CTX_EXECUTION_DATE=2022-06-22T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-22T00:00:00+00:00
[2022-06-27 13:33:31,918] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-27 13:33:31,919] {subprocess.py:74} INFO - Running command: ['bash', '-c', "echo 'accurate'"]
[2022-06-27 13:33:31,941] {subprocess.py:85} INFO - Output:
[2022-06-27 13:33:31,946] {subprocess.py:92} INFO - accurate
[2022-06-27 13:33:31,952] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-27 13:33:31,990] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=accurate, execution_date=20220622T000000, start_date=20220627T133331, end_date=20220627T133331
[2022-06-27 13:33:32,069] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:33:32,099] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
